src/AWS.Deploy.Common/IO/DirectoryManager.cs

```diff
@@ -61,5 +61,27 @@ public string[] GetProjFiles(string path)
         {
             return Directory.GetFiles(path).Where(filePath => _projFileExtensions.Contains(Path.GetExtension(filePath).ToLower())).ToArray();
         }
+
+        private bool IsDirectoryValid(string directoryPath)
+        {
+            directoryPath = directoryPath.Trim();
```

Comment:
Since this logic is the same as `IsFileValid` except for the Exists call can we put all of the string checks into a separate utility function that both methods call.

Reply:
Done